.github/workflows/home-view-unit-test.yaml

```diff
@@ -15,5 +15,3 @@ jobs:
           cache-dependency-path: "**/package-lock.json"
       - run: npm install
       - run: npm test
-      - name: Check Vitest thresholds
-        run: git diff --name-only --exit-code vite.config.ts
```

Comment:
FWIW I don't think this step is the issue, it fails on the one above if I'm reading correctly

Reply:
We encountered a failure earlier at this step, as the test step outputs a file w/ autoupdate, and then this step makes sure it is the same (and not showing as different).

Reply:
> FWIW I don't think this step is the issue, it fails on the one above if I'm reading correctly

Correct, removing the thresholds from the config removes the check against the thresholds preventing the failure. We no longer need this if we don't need to use the `coverage.thresholds.autoUpdate` feature to ensure we are keeping the config up-to-date.

Reply:
This has been removed since we are no longer using `autoUpdate` and we don't need to keep `vite.config.ts` up-to-date with the coverage threshold results.